tests/queries/0_stateless/03533_query_log_table_database_name_consistency_bug.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
```

Comment:
I think we can write this test as a `.sql` test instead of a `.sh` one. I prefer `.sql` tests over bash tests wherever we are able to do so.

Reply:
You're right, I did read that section of the documentation. However, I decided to write a bash test because I need a unique `log_comment` for each test run. This is because I create a database within the test itself to verify two things:

1. The database name in the `database` column.
2. The database name in the `tables` column.

For my tests, I create a database with a hyphen (`-`) in its name and then drop it after the test run is complete. I want to filter the logs to show only the results from a specific test run. As I understand it, I can achieve this by using either a `unique database name` or a `unique log_comment` for each run. However, I haven't found a way to generate a unique `log_comment` or `database name` from within the SQL test itself.